inventory/group_vars/all

```diff
@@ -1,5 +1,21 @@
+apache_ssl_use_stapling: yes
+
 hoist_repo: https://github.com/BonnyCI/hoist.git
 
+letsencrypt_account_key_content: "{{ secrets.letsencrypt.account_key }}"
+letsencrypt_csr_dn:
+  C: AU
+  ST: NSW
+  L: Australia
+  O: BonnyCI
+letsencrypt_configure_redirect: yes
+letsencrypt_email: jamielennox@gmail.com
+
+nodepool_mysql_host: zuul.bonnyci-internal.portbleu.com
```

Comment:
I think a non-prod environment could turn this feature off, by defining `letsencrypt_csr_cn` has blank?